registration-processor/registration-processor-packet-manager/src/test/java/io/mosip/registration/processor/packet/manager/service/impl/test/IdrepoDraftServiceTest.java

```diff
@@ -223,9 +256,43 @@ public void idrepoDraftReprocessableExceptionTest()
 				Lists.emptyList(), null, IdResponseDTO.class)).thenReturn(idResponseDTO);
 		when(registrationProcessorRestClientService.patchApi(any(), any(), any(), any(), any(), any()))
 				.thenReturn(idResponseDTO1);
+        ResponseDTO discardresponseDTO = new ResponseDTO();
+        discardresponseDTO.setStatus("Drafted");
+        discardresponseDTO.setRegistrationId(ID);
+
+        JSONObject jsonObject1 = new JSONObject();
+        jsonObject1.put("UIN", "1234");
+        discardresponseDTO.setIdentity(jsonObject1);
+        IdResponseDTO discardIdresponceDto=new IdResponseDTO();
+        discardIdresponceDto = new IdResponseDTO();
+        discardIdresponceDto.setErrors(null);
+        discardIdresponceDto.setId("id.uin.update");
+        discardIdresponceDto.setResponse(discardresponseDTO);
+        when(registrationProcessorRestClientService.
+                deleteApi(ApiName.IDREPODISCARDDREFT, Lists.newArrayList(ID), "", "", IdResponseDTO.class)).thenReturn(discardIdresponceDto);
 
 		idrepoDraftService.idrepoUpdateDraft(ID, null, idRequestDto);
+        Mockito.verify(idrepoDraftService.idrepodiscardDraft(any()),times(1));
 
 	}
-
+    @Test
+    public void discardDraftSuccessTest() throws IdrepoDraftReprocessableException, IdrepoDraftException, ApisResourceAccessException {
+        ResponseDTO discardresponseDTO = new ResponseDTO();
+        discardresponseDTO.setStatus("Drafted");
+        discardresponseDTO.setRegistrationId(ID);
+
+        JSONObject jsonObject1 = new JSONObject();
+        jsonObject1.put("UIN", "1234");
+        discardresponseDTO.setIdentity(jsonObject1);
+        IdResponseDTO discardIdresponceDto=new IdResponseDTO();
+        discardIdresponceDto = new IdResponseDTO();

```

Comment:
Same here,check two objects created for same object name and check for above points here

Reply:
changed.